examples/local/alias.source

```diff
@@ -0,0 +1,4 @@
+echo "Setting alias mysql to mysql -h 127.0.0.1 -P 15306"
```

Comment:
I believe this script should source `env.sh` too. That way we only need to have one `source` step in the docs, and can show the example code without a `source env.sh`.

Reply:
We could also do it the reverse, which might be more consistent with current usage.. and have env.sh source alias.source and then update the docs.

Reply:
Moving the alias to env.sh is causing spam. We could remove those messages, but we need to consider whether it will leave the users confused if we didn't inform them of the aliasing.

So, I'll skip this step for now, and we can hash this out more carefully before implementing.

Reply:
Sounds good. I'll merge this as soon as tests pass. It makes more sense in a separate PR anyway, since it is stepping out of helm and into local.